frontend/src/component/layout/MainLayout/NavigationSidebar/ShowHide.tsx

```diff
@@ -13,16 +13,16 @@ const ShowHideRow = styled(Box, {
     alignItems: 'center',
     padding: theme.spacing(0, 1, 0, mode === 'mini' ? 1.5 : 2),
     cursor: 'pointer',
-    position: 'sticky',
-    bottom: theme.spacing(2),
-    width: '100%',
 }));
 
 // This component is needed when the sticky item could overlap with nav items. You can replicate it on a short screen.
 const ShowHideContainer = styled(Box)(({ theme }) => ({
-    flexGrow: 1,
```

Comment:
instead of growing the show/hide container we moved this responsibility to the new Box that only occupies space